agent/agent-tooling/src/test/java/com/microsoft/applicationinsights/agent/internal/processors/ExporterWithAttributeProcessorTest.java

```diff
@@ -443,7 +443,7 @@ public void complexActionTest() {
     }
 
     @Test
-    public void simpleIncludeTest() {
+    public void simpleIncludeTest() throws FriendlyException {
```

Comment:
let's change `FriendlyException` to extend `RuntimeException` so we don't need to clutter up with all the throws clauses